gatsby-config.js

```diff
@@ -51,6 +51,10 @@ module.exports = {
             title: "Introduction to Journaling",
             path: "guides/journaling_intro.md",
           },
+          {
+            title: "App Builder Auto Subscriptions",
+            path: "guides/autosubscriptions.md",
+          },
```

Comment:
I think we should make this a child page of `Runtime Actions As Webhook` (guides/runtime_webhooks.md) as this is relevant for `App Builder / Runtime actions` integrations only. wdyt?